guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -0,0 +1,163 @@
+[id="Updating-Disconnected-{project-context}_{context}"]
+= Updating a Disconnected {ProjectServer}
+
+Update your air-gapped {Project} setup where the connected {ProjectServer}, which synchronizes content from CDN, is air gapped from a disconnected {ProjectServer}, to the next minor version.
+
+.Prerequisite
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+
+.Procedure on the connected {ProjectServer}
+
+. Ensure that you have synchronized the following repositories in your connected {ProjectServer}:
++
+* {RepoRHEL8BaseOS}
+* {RepoRHEL8AppStream}
+* {RepoRHEL8ServerSatelliteServerProductVersion}
+* {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+. Download the debug certificate of the organization and store it locally at `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
+For more information, see {AdministeringDocURL}Creating_an_Organization_Debug_Certificate_admin[Creating an Organization Debug Certificate] in _{AdministeringDocTitle}_.
+
+. Create a Yum configuration file under `/etc/yum.repos.d` with the following repository information:
++
+[options="nowrap" subs="attributes"]
+----
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/8/x86_64/baseos/os
+enabled=1
```

Comment:
By adding it in there you essentially make the server self-subscribed, which is something we don't support. If I read the help text right `--repo` (or `--repoid`) already enables it so I'd suggest to disable it by default.
```suggestion
enabled=0
```